http/headers/nel.json

```
@@ -0,0 +1,66 @@
+{
+  "http": {
+    "headers": {
+      "NEL": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/NEL",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "71"
+              },
+              {
+                "version_added": "69",
+                "flags": [
+                  {
+                    "type": "runtime_flag",
+                    "name": "Reporting",
+                    "value_to_set": "true"
+                  }
+                ]
+              }
+            ],
+            "chrome_android": {
+              "version_added": "71"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
```

```suggestion
              "version_added": "10.2"
```
…per the following:

https://github.com/mdn/browser-compat-data/blob/ee68a6df245329b16136f89fa69cd94627f9dc9e/browsers/samsunginternet_android.json#L150-L154